e2e/app/element/tab.ts

```diff
@@ -20,8 +20,10 @@ export default class Tab extends BaseElement {
 
   async isSelected(): Promise<boolean> {
     const element = await this.asElementHandle();
-    const selected = await getPropValue<boolean>(element, 'aria-selected');
-    return selected ? selected : !!(await getStyleValue<string>(this.page, element, 'border-bottom'));
+    const selected = await getPropValue<string>(element, 'ariaSelected');
+    return selected !== undefined
+      ? selected === 'true'
+      : (await getStyleValue<string>(this.page, element, 'border-bottom')) !== null;
```

Comment:
@jmthibault79 tab file updated with the help of Alex, it gets the aria-selected status. Now the tab-status is verification works as expected.